html/pages/settings/webui.inc.php

```diff
@@ -18,13 +18,21 @@
     ),
 );
 
+$dashboard_conf = array(
+    array('name'               => 'webui.default_dashboard_id',
+          'descr'              => 'Set global default dashboard id',
+          'type'               => 'text',
+    ),
+);
+
 echo '
 <div class="panel-group" id="accordion">
     <form class="form-horizontal" role="form" action="" method="post">
 ';
 
 echo generate_dynamic_config_panel('Graph settings',true,$config_groups,$graph_conf);
 echo generate_dynamic_config_panel('Search settings',true,$config_groups,$search_conf);
+echo generate_dynamic_config_panel('Dashboard settings',true,$config_groups,$dashboard_conf);
```

Comment:
@murrant: do you mean that I should change the 'true' to 'false'?  Or has the function been changed since I made this PR?